badger_api/requests.py

```diff
@@ -64,3 +67,13 @@ def fetch_all_claimable_balances(chain: str):
             data = future.result()["rewards"]
             results = {**results, **data}
     return results
+
+
+@lru_cache
```

Comment:
didn't we have an issue with caching before?

Reply:
This is caching for token names which has no effect on the rewards so this wouldn't cause any problems. Alternatively we could just create a global dict in the file but this is much cleaner.

Reply:
cool sounds good